stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -311,9 +311,15 @@ public struct ${Self}<Pointee>
   ///
   /// - Postcondition: The `Pointee`s at `self..<self + count` are
   ///   initialized.
+  // This is fundamentally unsafe since collections can underreport their count.
+  @available(*, deprecated, message: "it will be removed in Swift 4.0.  Please use 'UnsafeMutableBufferPointer.initialize(from:)' instead")
```

Comment:
Likewise, UnsafeRawPointer.initializeMemory should be moved to UnsafeRawBufferPointer!